### PR TITLE
feature/global_det_atmos_wmo_monthly_ops

### PR DESCRIPTION
**Pull request by malloryprow**
## Pull Request Testing ##

NCO said the monthly GFS WMO job can be run in operations. It will run every day but will only continue to generate the monthly report if VDATE is the end of the month.

To support that this PR:

1. Adds the monthly GFS WMO ecf script and to the evs-nco.def
2. Check in J-Job to determine if the WMO monthly reports should run.
3. Adds a commented out hold for DBN alerts for NCO to disseminate the GFS WMO monthly reports (technicalities need to be worked out with NCO)
4. Adds restart for the month's station information
5. Removes unneeded logging and repetitive export of environment variables.

- [ ] Describe testing already performed for this Pull Request:</br>
>I tested this branch for VDATE=20240731 and other various VDATEs that are not the end of the month.

- [ ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

- [ ] Has the code been checked to ensure that no errors occur during the execution? **Yes**

- [ ] Do these updates/additions include sufficient testing updates? **Yes**

- [ ] Please complete this pull request review by **8/26/2024**.</br>

## Pull Request Checklist ##

- [ ] Review the source issue metadata (required labels, projects, and milestone).
- [ ] Complete the PR description above.
- [ ] Ensure the PR title matches the feature branch name.
- [ ] Check the following:
- [ ]  Instructions provided on how to run
- [ ]  Developer's name is replaced by ${user} where necessary throughout the code
- [ ]  Check that the ecf file has all the proper definitions of variables
- [ ]  Check that the jobs file has all the proper settings of COMIN and COMOUT and other input variables
- [ ]  Check to see that the output directory structure is followed
- [ ]  Be sure that you are not using MET utilities outside the METplus wrapper structure

- [ ] After submitting the PR, select **Development** issue with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue.

### CODE REVIEW

**Comment by AliciaBentley-NOAA** [comment]
@malloryprow Quick and possibly dumb question: I see an ecf script in this PR, but not a dev driver. Should an accompanying dev driver be added to this PR? Maybe I'm just missing it?

CC @ShelleyMelchior-NOAA

**Comment by malloryprow** [comment]
> @malloryprow Quick and possibly dumb question: I see an ecf script in this PR, but not a dev driver. Should an accompanying dev driver be added to this PR? Maybe I'm just missing it?
> 
> CC @ShelleyMelchior-NOAA

The dev driver was put in with the original WMO PR #491 when I thought this portion would have to run in dev. Just needs an ecf now that is will be in ops.

**Comment by ShelleyMelchior-NOAA** [comment]
I have cloned your fork and checked out branch `feature/global_det_atmos_wmo_monthly_ops`.

File: scripts/stats/global_det/exevs_global_det_atmos_wmo_stats.sh [hunk 1]
```diff
@@ -74,3 +74,22 @@ if [ $SENDMAIL = YES ] ; then
...
+#        $DBNROOT/bin/dbn_alert MODEL EVS_WMO $job $COMOUT/${MODELNAME}.${VDATE}/202408_kwbc_monthly.rec2
+#        $DBNROOT/bin/dbn_alert MODEL EVS_WMO $job $COMOUT/${MODELNAME}.${VDATE}/202408_kwbc_dd10m.svs
+#        $DBNROOT/bin/dbn_alert MODEL EVS_WMO $job $COMOUT/${MODELNAME}.${VDATE}/202408_kwbc_ff10m.svs
+#        $DBNROOT/bin/dbn_alert MODEL EVS_WMO $job $COMOUT/${MODELNAME}.${VDATE}/202408_kwbc_rh2m.svs
+#        $DBNROOT/bin/dbn_alert MODEL EVS_WMO $job $COMOUT/${MODELNAME}.${VDATE}/202408_kwbc_t2m.svs
+#        $DBNROOT/bin/dbn_alert MODEL EVS_WMO $job $COMOUT/${MODELNAME}.${VDATE}/202408_kwbc_tcc.svs
+#        $DBNROOT/bin/dbn_alert MODEL EVS_WMO $job $COMOUT/${MODELNAME}.${VDATE}/202408_kwbc_tp06.svs
+#        $DBNROOT/bin/dbn_alert MODEL EVS_WMO $job $COMOUT/${MODELNAME}.${VDATE}/202408_kwbc_tp24.svs
+#    fi
+#fi
```
Comment (AliciaBentley-NOAA) [inline scripts/stats/global_det/exevs_global_det_atmos_wmo_stats.sh:95]: @malloryprow Just confirming that this section of WMO code satisfies this requested line from the EVS v2.0 Fixes and Additions document:

`Replace big comment block in WMO monthly aggregation code with DBNET alert`

Comment (malloryprow) [reply scripts/stats/global_det/exevs_global_det_atmos_wmo_stats.sh:95]: Yup, that be it!

**Comment by ShelleyMelchior-NOAA** [comment]
The first run for `jevs_global_det_gfs_atmos_wmo_monthly_stats.sh`, with no pre-set VDATE, has completed.

logfile: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_det_gfs_atmos_wmo_monthly_stats_00.o148930320`

The COMOUT directories are empty, which I believe is to be expected.
```
[clogin01 /lfs/h2/emc/ptmp/shelley.melchior/EVS_out]$ ls /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v2.0/stats/global_det/gfs.20240815/
[clogin01 /lfs/h2/emc/ptmp/shelley.melchior/EVS_out]$
[clogin01 /lfs/h2/emc/ptmp/shelley.melchior/EVS_out]$ ls /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v2.0/stats/global_det/atmos.20240815/gfs/wmo/
[clogin01 /lfs/h2/emc/ptmp/shelley.melchior/EVS_out]$
```

**Comment by malloryprow** [comment]
Yup to be expected since this test was to make sure it wasn't run and the correct message was displayed in the log 

> 1 + '[' 16 = 01 ']'
> 1 + echo 'Not generating monthly report'
> Not generating monthly report

**Comment by ShelleyMelchior-NOAA** [comment]
Great! I have kicked off the run that forces an export of VDATE=20240731. The process is underway.
logfile: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_det_gfs_atmos_wmo_monthly_stats_00.o148931930`

**Comment by ShelleyMelchior-NOAA** [comment]
The second run is complete! I see stats files for the month of July.
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v2.0/stats/global_det`

**Comment by malloryprow** [comment]
All good! The final reports match mine and log is good!

**Review by ShelleyMelchior-NOAA — APPROVED** [review]
The changes to the code test as expected. The PR is approved for merge.

**Comment by ShelleyMelchior-NOAA** [comment]
@malloryprow , did you want to change the 2nd reviewer to Perry from Alicia so that this PR can be merged before next week?

**Comment by malloryprow** [comment]
> @malloryprow , did you want to change the 2nd reviewer to Perry from Alicia so that this PR can be merged before next week?

Done! @PerryShafran-NOAA

**Comment by PerryShafran-NOAA** [comment]
Will do a quick code check and review.  Looks like all the testing is done?

**Review by PerryShafran-NOAA — APPROVED** [review]
All code looks good.  Approved for merge.

**Pull request merged**